sql_generators/glean_usage/templates/event_monitoring_live_v1.view.sql

```diff
@@ -1,9 +1,11 @@
-CREATE MATERIALIZED VIEW
-IF
-  NOT EXISTS `{{ project_id }}.{{ derived_dataset }}.event_monitoring_live_v1`
-  OPTIONS
-    (enable_refresh = TRUE, refresh_interval_minutes = 60) AS
-    {% if dataset_id not in ["telemetry", "accounts_frontend", "accounts_backend"] %}
+CREATE OR REPLACE MATERIALIZED VIEW
+  `{{ project_id }}.{{ derived_dataset }}.event_monitoring_live_v1`
+OPTIONS(
+  enable_refresh = TRUE,
+  refresh_interval_minutes = 60
+)
+AS
+  {% if dataset_id not in ["telemetry", "accounts_frontend", "accounts_backend"] %}
```

Comment:
I just noticed that if the `dataset_id` is "telemetry" then this will end up generating invalid SQL, because "telemetry" isn't included in the `elif` and there's no `else`.

Reply:
Yes, we intentionally don't want to monitor `telemetry`. This is for Glean apps only. Telemetry gets filtered out by default, since these views only get created for Glean apps, so we could just remove it from this list

Reply:
Since this PR is no longer making any other changes to that file's contents and it's essentially a non-issue, I'm going to leave it as is in this PR.